### ISSUE TITLE
alignment issues

### ISSUE THREAD

**Issue by DDR0** [al0fdf/frogatto-web-fix#1]
So, running through this, there seem to be some alignment issues. I've probably manually applied the patches wrong, but I can't figure out how.

![A picture of a rather mangled header, text overlapping poorly.](https://github.com/al0fdf/frogatto-web-fix/assets/862627/e4cf4577-89dd-4085-9c2a-b47dc4ad781d)

Here's the source file, although it's a bit rough to use outside of the framework. :thinking: 

[index.zip](https://github.com/user-attachments/files/16172398/index.zip)

I've fixed it up a bit myself, so it works, but I would like to move to maybe a fully flexbox solution or CSS grid?

**Comment by al0fdf** [comment]
> I've fixed it up a bit myself, so it works, but I would like to move to maybe a fully flexbox solution or CSS grid?

Probably a good choice going forward. I feel like the green download button and the brown download arrow being separate images makes it difficult to find a css setup that works to organize it.

**Comment by DDR0** [comment]
Yeah, they should probably be combined. I think they started out separate, but slowly migrated closer as the website progressed. :sweat_smile:
